### PR TITLE
New habit edge

### PR DESCRIPTION
**Pull request by pjconnolly12**
removed the route for new habit

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -30,10 +31,19 @@ class Dashboard extends Component {
...
+        habitData: [],
+        user: '',
     };
 
-    componentDidMount() {
+    componentDidMount = () => {
         this.updateProviders(this.state.providerData);
+        const user = this.state.providerData
+        this.setState({ user: user[0].email })
+        axios.get('/api/habits/habit/5cb0191dcca50e00d2e290be')
```
Comment (paulywill) [inline client/src/components/Dashboard.js:42]: I'm not familiar with axios.... what's happening on this line?

It that hardcoded for testing purposes?

**Review by paulywill — COMMENTED** [review]
It's functioning correctly locally.

Just what exactly is this doing?:

```axios.get('/api/habits/habit/5cb0191dcca50e00d2e290be')```

File: client/src/components/Dashboard.js (hunk 1 above)
Comment (pjconnolly12) [reply client/src/components/Dashboard.js:42]: That is the PR I pushed through for the GET request. I listed everything it's doing in that PR

**Review by paulywill — APPROVED** [review]
LGTM!

**Pull request merged**